SC Evo Complete/SCEvo_Core.SC2Mod/Base.SC2Data/GameData/UpgradeData.xml

```diff
@@ -156,262 +156,188 @@
         <AffectedUnitArray value="OverlordSCBW"/>
```

Comment:
the diff for this entire file is very messy due to using parents to inherit data, it's probably best viewed in the editor